starter-code/views/movie/edit.hbs

```diff
@@ -0,0 +1,11 @@
+<div>
+    <form action="/movies/{{mov._id}}" method="POST">
+        <label>Name:</label>
+        <input type="text" name="title" placeholder="Sharknado">
```

Comment:
Just one thing: in the edit view you should fill the input with the information of the objet to edit it. Use the `value={{mov.name}}` to do so... `<input type="text" name="title" placeholder="Sharknado" value={{mov.name}}>`